### DISCUSSION TITLE
loading symbols

### DISCUSSION THREAD

**Discussion by satvik-1203** [community/community#8980] [Projects and Issues]
I'm not sure why this is happening, I thought GitHub was majorly SSR so loading symbols are rare to see and they don't go away even after 20 secs+.

<img width="272" alt="Screen Shot 2021-12-17 at 6 17 15 AM" src="https://user-images.githubusercontent.com/77904774/146470291-73b62479-6772-44e5-9ab0-b7406e537dce.png">

<img width="128" alt="Screen Shot 2021-12-17 at 6 17 37 AM" src="https://user-images.githubusercontent.com/77904774/146470314-f0b5fc19-0ba1-439f-9d1f-00de9da07de7.png">

**Comment by Deadlyelder** [comment]
It is also seen within the open PR description especially if the main post for the PR has been edited.

EDIT: Also seen in organization pages

<img width="268" alt="Screen Shot 2021-12-17 at 7 57 33 AM" src="https://user-images.githubusercontent.com/6140893/146502648-575c2f54-6f37-4cc9-9679-61fb52aabda4.png">
